svn_extractor.py

```diff
@@ -8,13 +8,18 @@
 import traceback
 import re
 
+def getext(filename):
+    name, ext = os.path.splitext(filename)
+    return ext[1:]
```

Comment:
i suppose this should be [:-1] to capture last element only and not the entire portion after first element. Example if file name is XYZ.exe.jpg currently the getext will get "exe.jpg" we want "jpg"